Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+fhBuildNode() {
+    stage('Install Dependencies') {
+        npmInstall {}
+    }
+
+    stage('Lint') {
+        sh 'grunt eslint'
+    }
+
+    stage('Unit Tests') {
+        withOpenshiftServices(['mongodb']) {
+            sh "node ./test/setup_mongo_wendy.js"
+            sh 'grunt fh:test'
```

Comment:
`grunt fh:test` will run all kinds of tests -- unit/acceptance/integration. The stage name here just says unit tests. You can either run unit tests with `grunt fh:unit`, then run acceptance tests with `grunt fh:accept`; or change the stage name.